FinanceTracker/FinanceTracker/Models/Operation.cs

```diff
@@ -1,22 +1,32 @@
 ﻿using FinanceTracker.Enums;
+using System.Text.Json.Serialization;
 
 namespace FinanceTracker.Models
 {
     public class Operation
     {
-        public OperationType Type { get; set; }
+        [JsonPropertyName("id")]
         public Guid ID { get; set; }
+
+        [JsonPropertyName("type")]
```

Comment:
To avoid putting this everywhere, you can try using camelCase naming policy on serializer level
```
var serializeOptions = new JsonSerializerOptions
{
    PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
    WriteIndented = true
};
jsonString = JsonSerializer.Serialize(weatherForecast, serializeOptions);
```
https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?pivots=dotnet-8-0